lib/types/eacp.js

```diff
@@ -0,0 +1,62 @@
+/*!
+ * Tozny e3db
+ *
+ * LICENSE
+ *
+ * Tozny dual licenses this product. For commercial use, please contact
+ * info@tozny.com. For non-commercial use, this license permits use of the
+ * software only by government agencies, schools, universities, non-profit
+ * organizations or individuals on projects that do not receive external
+ * funding other than government research grants and contracts. Any other use
+ * requires a commercial license. For the full license, please see LICENSE.md,
+ * in this source repository.
+ *
+ * @copyright Copyright (c) 2018-19 Tozny, LLC (https://tozny.com)
+ */
+
+'use strict'
+
+import Serializable from './serializable'
+import EmailEACP from './emailEACP'
+import NoteAccessEACP from './noteAccessEACP'
+
+/**
+ * NoteOptions represents optional values that are not required for creating a note,
```

Comment:
same copy pasta comment

Reply:
yeah, looks like I need to do a code-comment pass. That's been killing me lately.